avocado/core/settings.py

```diff
@@ -171,10 +173,15 @@ def __init__(self, config_path=None):
             config_local = os.path.exists(config_path_local)
             config_intree = os.path.exists(config_path_intree)
             config_intree_extra = os.path.exists(_config_path_intree_extra)
-            if (not config_system) and (not config_local) and (not config_intree):
+            config_pkg_base = os.path.join('etc', config_filename)
+            config_pkg_exists = resource_exists('avocado', config_pkg_base)
+            config_pkg = resource_filename('avocado', config_pkg_base)
+            if not (config_system or config_local or
+                    config_intree or config_pkg_exists):
```

Comment:
As mentioned in the previous PR would you please adjust the variable names according the usual scheme (`config_$name` and `config_path_$name`)?

Reply:
Sure, changing that on v3.